recipes/gmp/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+chmod +x configure
+
+./configure --prefix=$PREFIX --enable-cxx
```

Comment:
For some reason the original recipe only built C++ support in some cases. This feels like a weird restriction and has been removed.